src/charts/onPreprocess/hideNormalRangeControls.js

```diff
@@ -0,0 +1,10 @@
+export default function hideNormalRangeControls() {
+    //console.log(
```

Comment:
remove?

Reply:
Yup, handling this in the parent object rather than in the multiples'.